src/yunohost/alias.py

```diff
@@ -0,0 +1,186 @@
+# -*- coding: utf-8 -*-
+
+""" License
+
+    Copyright (C) 2016 YUNOHOST.ORG
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program; if not, see http://www.gnu.org/licenses
+
+"""
+
+""" yunohost_alias.py
+
+    Manage alias
+"""
+
+import errno
+
+from moulinette.core import MoulinetteError
+from yunohost.domain import domain_list
+
+def alias_list(auth):
+    """
+    List aliases
+
+    """
+    _ensure_ldap_ou_is_created(auth)
```

Comment:
Cannot agree more, but before such migration framework is merged, it does the trick quite well/transparently.